src/components/Feed/GlobalFeed.tsx

```diff
@@ -0,0 +1,76 @@
+import { useCallback } from 'react';
+import { graphql, useLazyLoadQuery, usePaginationFragment } from 'react-relay';
+import styled from 'styled-components';
+import { GlobalFeedQuery } from '__generated__/GlobalFeedQuery.graphql';
+import { WORLDWIDE } from './Feed';
+
+import FeedList from './FeedList';
+
+export default function GlobalFeed() {
+  const ITEMS_PER_PAGE = 10;
+  const query = useLazyLoadQuery<GlobalFeedQuery>(
+    graphql`
+      query GlobalFeedQuery($before: String, $last: Int) {
+        ...GlobalFeedFragment
+        ...FeedEventQueryFragment
+      }
+    `,
+    {
+      last: ITEMS_PER_PAGE,
+    }
+  );
+
+  const { data, hasPrevious, loadPrevious, isLoadingPrevious } = usePaginationFragment<
+    GlobalFeedQuery,
+    any
```

Comment:
oh so the relay docs said this one doesn't have to be specified, so it suggested using `_` so that the type can be inferred, but type check didn't like it so I just threw `any` in here
https://relay.dev/docs/api-reference/use-pagination-fragment/

do you know if we can use `_`?

Reply:
ah gotcha, any should be chill then!